internal/controllers/database/init.go

```diff
@@ -62,9 +63,60 @@ func (r *Reconciler) setInitDatabaseCompleted(
 		Reason:  ReasonCompleted,
 		Message: message,
 	})
+	meta.SetStatusCondition(&database.Status.Conditions, metav1.Condition{
+		Type:   CreateDatabaseOperationCondition,
+		Status: metav1.ConditionTrue,
+		Reason: ReasonCompleted,
+	})
 	return r.updateStatus(ctx, database, StatusUpdateRequeueDelay)
 }
 
+func (r *Reconciler) checkCreateTenantOperation(
+	ctx context.Context,
+	database *resources.DatabaseBuilder,
+	tenant *cms.Tenant,
+	ydbOptions ydb.Option,
+) (bool, ctrl.Result, error) {
+	condition := meta.FindStatusCondition(database.Status.Conditions, CreateDatabaseOperationCondition)
+	if condition == nil || len(condition.Message) == 0 {
+		meta.RemoveStatusCondition(&database.Status.Conditions, CreateDatabaseOperationCondition)
```

Comment:
For what we should remove condition? Just keep it for history

Reply:
In case when need to retry create tenant, the condition CreateDatabaseOperation will be set True
In my opinion, It looks strange

Anyway, I did as you asked